README.md

```diff
@@ -1,80 +1,201 @@
+around the donut and fits a model using astropy.modeling that has one 
+value in the "hole" of the  donut, one value for the value in the 
+pupil plate and a background level. The centers and radii of the circles 
+that form the boundaries of the pupil plate are also allowed to vary. 
+There is code in this repo to fit ellipses instead of circles, and to fit
+brightness gradient across the donut, but they are currently disabled.
+
+The results are saved in the output directory specified by the user at runtime.
+Each star in each image that is fit as a pupil plate produces a pdf file
+with the filename of the image followed by a source id. At the end of the
+analysis the pdfs are merged by defocus position into
+pupe-pat-{focdmd}-donut.pdf (default) where {focdmd} is replaced 
+with the requested focus of the images. The code also produces a "quiver"
+plot (replacing donut with quiver in the name above). This shows the magnitude
+and direction of the offset of the centers of the inner and outer edges of
+the pupil plate. The "best" M2 tip and tilt are marked with a red X.
+
+The code can be run in two modes (see below): real-time and offline modes.
+The offline mode will analyze all images in a given directory that are 
+taken by the LCOEngineering proposal (default), that are EXPOSE or EXPERIMENTAL
+obstypes, and are defocused (FOCDMD != 0.0). To ensure that the proposal
+is correctly included in the header, the Sequencer should be set to manual
+when taking the images.
+
+The real-time mode has the same criteria for which images to analyze, but
+watches the input directory for new files. Right before an M2 tip/tilt run,
+this watcher should be started on the directory where the files will appear. 
+For the sbig cameras, this can be the `raw` directory for the given DAY-OBS.
+The Sinistro frames need the amplifiers to be stitched together before they
+can be analyzed, so the watcher should be directed to the `preview` directory
+instead.
+
+[//]: # (Code examples of the usage)
+## Examples
+The standard offline analysis can be run using the following command.
+```
+docker pull docker.lco.global/pupe-pat:{0.1.0}
+docker run --rm -it -v /home/eng/pupe-pat-test-data:/input -v /home/eng/pupe-pat-tests-{0.1.0}:/output docker.lco.global/pupe-pat:{0.1.0} run_pupepat --output-dir /output --input-dir /input
+```
+Again replace `{0.1.0}` with the current version. This gets the most 
+recent version and starts up a docker container
+(which is kind of like a virtual machine).
+The `--rm` flag removes the container when it is finished to keep the system
+cleaned up. The `-it` runs the container in interactive mode. This should 
+not be strictly necessary. The `-v` maps a directory on the current machine
+into a directory in the docker container. The path on the left should be wherever
+the raw data is and where you want the output results to go. The argument on the 
+right of the `:` can be anything as long as the `--input-dir` and `output-dir`
```

Comment:
Sorry, I can't make head nor tail of the path discussion

Reply:
Hopefully have the single shell script alleviates most of this discussion.